taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.http import HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Manufacturer, Car
+
+
+def index(request) -> HttpResponse:
+    num_drivers = get_user_model().objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
```

Comment:
You need to follow a consistent style. You have three variables: two declared using class syntax and one using `get_user_model()`. While this solution will also work, it's better if all three variables are declared in the same way.